components/card/card.js

```diff
@@ -1,7 +1,6 @@
 export function createCharacterCard(url, status, type, occurrences) {
-  //const card = document.querySelector("[data-js=card-container]");
-
   const li = document.createElement("li");
```

Comment:
I think, you can find a more descriptive name for this variable, e.g. `listItem` / `card` etc. instead of just `li`.